libuavcan/src/transport/can_acceptance_filter_configurator.cpp

```diff
@@ -0,0 +1,256 @@
+/*
```

Comment:
Please rename the file to `uc_can_acceptance_filter_configurator.cpp` (all other `.cpp` files have prefixed names, this is needed to avoid name collisions on certain build systems).